extension/task/task.json

```diff
@@ -40,6 +45,28 @@
     }
   ],
   "inputs": [
+    {
+      "name": "command",
+      "type": "pickList",
+      "label": "Command",
+      "options": {
+        "update_script": "Update dependencies",
+        "update_script_vnext": "Update dependencies (vNext)",
+        "custom": "Custom"
+      },
+      "defaultValue": "update_script",
+      "required": true,
+      "helpMarkDown": "The Dependabot command to run. Select 'Custom' to use a command not listed here."
+    },
+    {
+      "name": "customCommand",
+      "type": "string",
+      "label": "Custom Command",
+      "defaultValue": "",
+      "required": false,
+      "helpMarkDown": "The custom command to run. This is only used when the command is set to 'Custom'.",
+      "visibleRule": "command=custom"
+    },
     {
```

Comment:
Let us make this just a boolean input to choose between `updater_script` and `updater_vnext`

Reply:
done